client/components/wcc-settings-form/index.js

```diff
@@ -24,4 +47,37 @@ WCCSettingsForm.propTypes = {
 	saveFormData: PropTypes.func.isRequired,
 };
 
-export default WCCSettingsForm;
+const getFormErrors = ( schema, data ) => {
+	const validate = validator( schema, { greedy: true } );
+	const success = validate( data );
+
+	if ( ! success && validate.errors && validate.errors.length ) {
```

Comment:
`is-my-json-valid` prepends all errors (paths to fields) with `data`, so this map removes that.